ui/src/components/role/RoleTable.js

```diff
@@ -155,6 +155,7 @@ export default class RoleTable extends React.Component {
                     // group rows
                     let roleGroup = (
                         <RoleGroup
+                            key={'group:' + name}
```

Comment:
what is this for?

Reply:
To add the unique key id for Role Group element

Reply:
There is a error reporting in the console